versions/index.html

```diff
@@ -1,5 +1,6 @@
 ---
 layout: fullwidth
+omit_from_search: true
```

Comment:
is this the way that you prevent a page being indexed? We should document this somewhere

Reply:
Yes it is. I added it to the indexer plugin's inline documentation.

Reply:
I think we should be more explicit than that - can we throw it in the `README.md` file?